speckle/Cache.py

```diff
@@ -42,14 +42,29 @@ def __init__(self, filepath=None):
             filepath {str} -- Optional database filepath (default: {None})
         """        
         if filepath is None:
-            self.db_path = os.path.join(os.getenv('LOCALAPPDATA'), os.path.join('SpeckleSettings', 'SpeckleCache.db'))
+
+            """
+            Get platform-dependent default SpeckleSettings directory
+            """
+            settings_dir = ""
+            if platform.system() == "Windows":
+                settings_dir = os.path.join(os.getenv('LOCALAPPDATA'), 'SpeckleSettings')
+            elif platform.system() == "Darwin":
```

Comment:
I would use `else` here as any other system is going to be Unix based and therefore use the "Home" env var

Reply:
Cool! I have very little experience with variables on other platforms :)